testdata/a_bit_of_everything.plist

```diff
@@ -0,0 +1,61 @@
+<TestMe>
```

Comment:
to be a valid plist this should have the XML preamble etc; but of course, we aren't really deserializing an actual plist, or we wouldn't have this problem.

I avoided having to think about this by just writing the strings for my test data inline in the functions, so maybe you could do the same? Then we could also reuse the same test data for both ser&de.

Reply:
Ah okay I hadn't caught that these files are auto-generated. In that case I would just use the `.xml` suffix, and then as follow-up maybe we can reuse these files for the parsing test.